gcn/notices/fermi/gbm/strict/alert.schema.json

```diff
@@ -0,0 +1,168 @@
+    },
+    "alert_tense": {
+      "enum": [
+        "current",
+        "archival",
+        "planned",
+        "injection",
+        "commanded",
+        "test"
+      ],
+      "default": "current",
+      "description": "Indication of whether this alert refers to a recent observation (current), re-analysis of archival data (archival), a planned observation in the future (planned), a signal injection (injection), commanded trigger (commanded) or a test trigger (test)."
+    },
+    "alert_type": {
+      "enum": ["initial", "update", "retraction"],
+      "default": "initial",
+      "description": "Indication of alert sequence if multiple of same type sent"
+    },
+    "data_archive_page": {
+      "type": "string",
+      "description": "URL of archived data files"
+    },
+    "detectors_status": {
+      "type": "array",
+      "items": {
+        "enum": ["on", "off", "triggered"]
+      },
+      "minItems": 14,
+      "maxItems": 14,
+      "description": "Triggered detectors"
+    },
+    "event_name": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      },
+      "description": "Name of the event, for example GRB170817A"
+    },
+    "hitl": {
+      "type": "boolean",
+      "default": false,
+      "description": "\"Human in the loop\".  When false, the notice was generated entirely automatically.  A value of true means that a human had an input, while the notice was being prepared.  The latter includes notices, which weren't altered by the human input, i.e. simply confirmed or approved by a human for further processing."
+    },
+    "id": {
+      "type": "array",
+      "items": {
+        "oneOf": [
+          {
+            "type": "string"
+          },
+          {
+            "type": "number"
+          }
+        ]
+      },
+      "description": "Instrument-specific trigger ID, for example: 745121685, bn240812094"
+    },
+    "instrument": {
+      "const": "GBM",
+      "default": "GBM",
+      "description": "Name of the instrument reporting the event"
+    },
+    "latitude": {
+      "type": "number",
+      "description": "Geocentric latitude [deg]"
+    },
+    "light_curve_url": {
+      "type": "string",
+      "description": "URL of the quicklook light curve plot"
+    },
+    "longitude": {
+      "type": "number",
+      "description": "Geocentric longitude [deg]"
+    },
+    "messenger": {
+      "const": "EM",
+      "default": "EM",
+      "description": "Messenger of report; EM, GW or Neutrino"
+    },
+    "mission": {
+      "const": "Fermi",
+      "default": "Fermi",
+      "description": "Name of Mission or Telescope reporting the event"
+    },
+    "notice_type": {
+      "const": "FERMI_GBM_ALERT",
+      "description": "Notice title"
+    },
+    "rate_duration": {
+      "type": "number",
+      "description": "Duration in seconds of the ime interval used in the signal to noise ratio calculation"
```

Comment:
```suggestion
      "description": "Duration in seconds of the time interval used in the signal to noise ratio calculation"
```

Reply:
@Vidushi-GitHub: Typo fixed. Thanks for the catch!